### PR TITLE
 [MIG] hr_employee_lastnames: Migration to 14.0

### PR DESCRIPTION
**Pull request by luistorresm**

### CODE REVIEW

**Comment by luistorresm** [comment]
Hello @moylop260 

Could you review and merge this PR if you agree, please?

Regards

File: hr_employee_lastnames/README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/13.0/hr_employee_lastnames
+    :alt: OCA/hr
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-13-0/hr-13-0-hr_employee_lastnames
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/13.0
```
Comment (moylop260) [inline hr_employee_lastnames/README.rst]:
```suggestion
    :target: https://runbot.odoo-community.org/runbot/116/14.0
```

File: hr_employee_lastnames/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/13.0/hr_employee_lastnames
+    :alt: OCA/hr
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-13-0/hr-13-0-hr_employee_lastnames
```
Comment (moylop260) [inline hr_employee_lastnames/README.rst]:
```suggestion
    :target: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_employee_lastnames
```

File: hr_employee_lastnames/README.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/hr <https://github.com/OCA/hr/tree/13.0/hr_employee_lastnames>`_ project on GitHub.
```
Comment (moylop260) [inline hr_employee_lastnames/README.rst]:
```suggestion
This module is part of the `OCA/hr <https://github.com/OCA/hr/tree/14.0/hr_employee_lastnames>`_ project on GitHub.
```

**Review by moylop260 — CHANGES_REQUESTED** [review]
Please, check the `s/13.0/14.0/g` comments, please

File: hr_employee_lastnames/README.rst (hunk 3 above)
Comment (emagdalenaC2i) [reply hr_employee_lastnames/README.rst]: @moylop260 the README file is autogenerated after merge

Comment (luistorresm) [reply hr_employee_lastnames/README.rst]: Thanks @emagdalenaC2i

@moylop260 

Was applied.

Regards

**Pull request merged**